packages/rpc/src/chainStorageWatcher.ts

```diff
@@ -201,10 +201,30 @@ export const makeAgoricChainStorageWatcher = (
     return () => stopWatching(pathKey, subscriber as Subscriber<unknown>);
   };
 
+  const queryOnce = <T>(path: [AgoricChainStoragePathKind, string]) =>
```

Comment:
remark: nice use of generic. eventually I'd like agoric-sdk to export a schema of types for each storage path so that the type can be inferred from the path string.